pkg/controller/jobframework/setup.go

```diff
@@ -92,6 +92,11 @@ func (m *integrationManager) setupControllers(ctx context.Context, mgr ctrl.Mana
 						log.Error(err, "Failed to setup controller and webhook for job framework")
 					}
 				})
+				// Webhook must be registered now.
```

Comment:
Please also adjust the logging, the previous `logger.Info("No matching API in the server for job framework, skipped setup of controller and webhook")` is no longer true.

Reply:
Done